Import/index.mjs

```diff
@@ -0,0 +1,61 @@
+import {Manifest} from "manifesto.js"
+import dbDriver from "../database/driver.mjs"
+
+const database = new dbDriver("mongo")
+
+export class Project {
+  constructor(data) {
+    this.data = data
+  }
+
+  async create(payload) {
+    return database
+      .save({...payload, "@type": "Project"})
+      .then((savedProject) => {
+        return savedProject
+      })
+  }
+
+  static async fetchManifest(manifestId) {
+    const url = `https://t-pen.org/TPEN/project/${projectId}`
+    return fetch(url).then((response) => {
+      return response.json()
+    })
+  }
+
+  static async processManifest(manifestId) {
+    return Project.fetchManifest(manifestId)
+      .then((manifest) => {
+        let newProject = {}
+        newProject.title = manifest.title
+        newProject.label = manifest.label
+        newProject.metadata = manifest.metadata
+        newProject.pages = manifest.items ?? manifest.sequences
```

Comment:
Exactly right, this is all you can really get from the Manifest, these are each of the pieces.

This is where the TPEN 3 Pages come from.  However, `manifest.items` and `manifest.sequences` do not contain TPEN 3 Pages, they contain Canvases.  You will have to import classes/Page/Page.mjs and be prepared to use that to process Canvases.

So you will need to loop those and make TPEN 3 Pages (which may contain Layers and Lines based off the Canvas AnnotationList or AnnnotationPage).  Then you can associated those TPEN 3 Pages with this TPEN 3 Project.

```javascript
//manifest.items and manifest.sequences have a different structure, so be careful with how you loop it
const canvases = getCanvasesFromManifest(manifest)
newProject.pages = canvases.map(canvas => {
    // createPage will also process the Annotations into Layers/Lines
	return Page.createPage(canvas)
})
```